rough_draft_aim2.md

```diff
@@ -0,0 +1,17 @@
+
+Human Cell Atlas' partnership with the Immunological Genome Project (immgenH) will result in gene expression-based immunocyte phenotyping at unprecedented resolution, making it well-suited for the problem of studying rheumatic/autoimmune disorders and for addressing the question of the interplay between tissue-resident immune cells and those cells infiltrating from peripheral blood in these disease states.
```

Comment:
The sentence also seems to run a bit long - maybe split into two. This is a rough draft tho, so I'll focus most of my comments on clarity of approach